etc/kconfigs.yaml

```diff
@@ -284,7 +284,7 @@ WW_MUTEX_SELFTEST: v4.11-rc1
 X86_CPU_RESCTRL: v5.0-rc1
 X86_INTEL_TSX_MODE_AUTO: v5.4-rc1
 X86_INTEL_TSX_MODE_OFF: v5.4-rc1
-X86_INTEL_TSX_MODE_ON:: v5.4-rc1
+X86_INTEL_TSX_MODE_ON: v5.4-rc1
```

Comment:
the patch subject need be like etc/kconfigs.yaml: fix typo of X86_INTEL_TSX_MODE_ON::

Reply:
just a test to see if my comments show